Application/Application.csproj

```diff
@@ -15,6 +15,7 @@
 
     <ItemGroup>
         <ProjectReference Include="..\Domain\Domain.csproj"/>
+
```

Comment:
Ovakve promene bez značenja prave "noise" - to obično možeš da uloviš u Changes pre nego što odradiš commit, pogledaš svaki fajl šta je menjano, i ako je ovakva promena samo ideš rollback za taj fajl